reference/halyard/high-availability.md

```diff
@@ -135,6 +135,14 @@ This service's name when [configuring its sizing](/reference/halyard/component-s
 
 To add a [custom profile](/reference/halyard/custom/#custom-profiles) or [custom service settings](/reference/halyard/custom/#custom-service-settings) for this service, use the name `echo-worker`.
 
+## Deleting Orphan Services
+
+When enabling or disabling HA for a service on a running Spinnaker, Halyard will not clean up the old service(s) by default. This means that if a non-HA Clouddriver is running and Spinnaker is then deployed with HA Clouddriver enabled, the non-HA Clouddriver will still be running, even though it is no longer used. To clean up these orphan services, add a `--delete-orphan-services` flag to `hal deploy apply`:
```

Comment:
But not just Clouddriver, right? Can be any service? If so, I would add a "for example" to the above:

"This means that if a non-HA Clouddriver is running (for example)  and Spinnaker is then deployed..."